src/scripts/components/widgets/modals/ModalAskPassword.js

```diff
@@ -76,37 +82,41 @@ class ModalAskPassword extends Component<Props, Object> {
       { title: 'Trying to unlock your keystore...' },
       'warning'
     )
-    try {
-      paratii.eth.wallet.clear()
-      paratii.eth.wallet.decrypt(JSON.parse(walletString), password)
-      this.props.notification(
-        {
-          title: 'Success!',
-          message: 'Your keystore has been unlocked.'
-        },
-        'success'
-      )
-      // Set the balance
-      this.props.setAddressAndBalance()
-      this.props.setWalletData({ walletKey: WALLET_KEY_SECURE })
-      /// Update user data in redux state
-      this.props.setUserData()
-      // Retrieve your videos
-      this.props.fetchOwnedVideos()
-      this.props.closeModal()
-    } catch (err) {
-      // Password is not valid
-      this.setState({
-        error: 'The password is not valid, please retype the password'
-      })
-      this.props.notification(
-        {
-          title: 'The password is not valid',
-          message: 'Please retype the password.'
-        },
-        'error'
-      )
-    }
+
+    setTimeout(() => {
```

Comment:
This timeout is necessary to show the notification "Trying to unlock your keystore...". We are doing this elsewhere in the app